### ISSUE TITLE
Pre-grooming issue review (10/14)

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#14451]
## Issue Description
Some issues need time to review before grooming but do not need a complete discovery effort. Due to lack of familiarity, this is particularly true for VAMC template issues and some 508/accessibility changes.

The following issues need to be reviewed so that Facilities team engineers have a basic understanding of the code and feel prepared to groom on 10/14:

**Front end engineering**
- [x] [#14220](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14220) "Not applicable" value for facility health service
- [x] [#14578](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14578) FE implementation of CMS new date/time zone fields 
- [x] #6009  Make it possible to view tables (on VAMC pages) at 400% zoom levels
- [x] [#14018](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14018) Change path for /v0/facilities/va 

**Back end engineering**
- [ ] [#14018](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14018) Change path for /v0/facilities/va

---
## Tasks
- [ ] **Quick** code review/familiarize yourself with the issue described  
- [ ] Identify dependencies
- [ ] Identify questions for other teams

## Acceptance Criteria
- [ ] Engineers are prepared to groom the issue
- [ ] Unknowns are identified
- [ ] Dependenices are identified
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mpelzsherman** [comment]
@darkbushido Please have a look at #14018 and see if there's any BE work needed, thanks!
